sdk/core/azure-core/src/samples/java/com/azure/core/annotation/AnnotationJavadocCodesnippets.java

```diff
@@ -335,7 +335,7 @@ void restart(@PathParam("resourceGroupName") String rgName,
 
         // BEGIN: com.azure.core.annotation.Post.class2
         @Post("https://{functionApp}.azurewebsites.net/admin/functions/{name}/keys/{keyName}")
-        NameValuePair generateFunctionKey(@PathParam("functionApp") String functionApp,
+        KeyValuePair generateFunctionKey(@PathParam("functionApp") String functionApp,
```

Comment:
Should we include `KeyValuePair` coming from wiremock dependency in code samples?

Reply:
Probably not, I did this for convenience and to keep the sample roughly the same.